### ISSUE TITLE
Friend Notifications Plugin

### ISSUE THREAD

**Issue by Ahmed-2987** [Vencord/plugin-requests#12]
### Plugin Description

A plugin that shows when a friend goes online, offline, idle, DND mode or changes status, etc... Could be customized to choose who you want to be notified about. 

### Existing Plugin for other mod

https://betterdiscord.app/plugin/FriendNotifications

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Roki100** [comment]
yes pls

**Comment by D3SOX** [comment]
I already did that locally. Can push as a PR later :)

**Comment by EdVraz** [comment]
https://github.com/Vendicated/Vencord/pull/2175

**Comment by KontonChowa** [comment]
any news on this plugin being finished/added?

**Comment by practiceineverything** [comment]
> I already did that locally. Can push as a PR and userplugin later :)

I wonder if this plugin will ever be released

**Comment by zLoveMyyScxF** [comment]
yeah i hope this will get released soon, would be such a nice feature ngl
